docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -103,4 +121,14 @@ Prisma Cloud now seamlessly identifies 'Rejected' and 'Disputed' statuses of CVE
 |*Fixed Containerized Scan Failure*
 |Resolved issue causing containerized scans to fail due to long JSON, particularly when encountering large Java dependency lists. You can now conduct scans without encountering this issue.
 
+//CWP-56788
+//TODO:Pending blurb approval for Kamesh
+|*Defender fails to re-scan TAS applications after changes*
+|Previously, if modifications were done to a TAS application (such as renaming it), Defender would fail to re-scan the application and update the results in the Console. This issue has been resolved, and Defender now re-scans the TAS applications whenever changes are made to them.
```

Comment:
This issue has been resolved, and Defender now re-scans the TAS applications whenever changes are made.